src/cmd/linuxkit/Makefile

```diff
@@ -54,6 +54,11 @@ ifeq ($(STATIC), 1)
 	CGO_ENABLED=0
 endif
 
+# darwin needs CGO to build for virtualization framework
+ifeq ($(GOOS), darwin)
```

Comment:
does this still produce static binaries? do we not need `--tags netgo` to use Go's network stack and not the libc one?

Reply:
I think go always uses its own network stack. The question is what dynamic libraries it loads.

To be fair, even prior to this it wasn't 100% static. Here is the relevant output from `otool -o linuxkit` off of current master branch:
```
Load command 11
          cmd LC_LOAD_DYLINKER
      cmdsize 32
         name /usr/lib/dyld (offset 12)
Load command 12
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libSystem.B.dylib (offset 24)
   time stamp 0 Thu Jan  1 02:00:00 1970
      current version 0.0.0
compatibility version 0.0.0
Load command 13
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (offset 24)
   time stamp 0 Thu Jan  1 02:00:00 1970
      current version 0.0.0
compatibility version 0.0.0
Load command 14
          cmd LC_LOAD_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/Security.framework/Versions/A/Security (offset 24)
   time stamp 0 Thu Jan  1 02:00:00 1970
      current version 0.0.0
compatibility version 0.0.0
```

And from the version off of this branch:

```
Load command 12
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libobjc.A.dylib (offset 24)
   time stamp 2 Thu Jan  1 02:00:02 1970
      current version 228.0.0
compatibility version 1.0.0
Load command 13
          cmd LC_LOAD_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation (offset 24)
   time stamp 2 Thu Jan  1 02:00:02 1970
      current version 1856.105.0
compatibility version 300.0.0
Load command 14
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/Virtualization.framework/Versions/A/Virtualization (offset 24)
   time stamp 2 Thu Jan  1 02:00:02 1970
      current version 59.4.0
compatibility version 1.0.0
Load command 15
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libSystem.B.dylib (offset 24)
   time stamp 2 Thu Jan  1 02:00:02 1970
      current version 1311.0.0
compatibility version 1.0.0
Load command 16
          cmd LC_LOAD_DYLIB
      cmdsize 104
         name /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (offset 24)
   time stamp 2 Thu Jan  1 02:00:02 1970
      current version 1856.105.0
compatibility version 150.0.0
Load command 17
          cmd LC_LOAD_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/Security.framework/Versions/A/Security (offset 24)
   time stamp 2 Thu Jan  1 02:00:02 1970
      current version 60157.60.19
compatibility version 1.0.0
```

Not all that different, adds these:

```
/usr/lib/libobjc.A.dylib
/System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
/System/Library/Frameworks/Virtualization.framework/Versions/A/Virtualization
```

which probably is just virtualization plus its upstream dependencies.

I am fine with it the way it is.

Reply:
@dave-tucker I need you to approve these changes so I can merge it in